apps/screloc/descriptions/screloc.xml

```diff
@@ -72,6 +72,19 @@
 					</description>
 				</parameter>
 			</group>
+			<group name="picks">
+				<description>
+				Parameters specific to picks.
+				</description>
+				<parameter name="streamsUnused" type="list:string" default="">
```

Comment:
I am not convinced by the name "streamsUnused". It sounds like it has to do with streams that are already unused. I would call it "ignoreStreams" or something that indicates the purpose.

Reply:
The streams are not ignored but set to unused. Who knows, what locators do with those. How about calling the parameter "streamsSetUnused"?

Reply:
Good point that the locator might still use those `unused` streams